packages/go_router_builder/pubspec.yaml

```diff
@@ -0,0 +1,34 @@
+name: go_router_builder
+description: >-
+  A builder that supports generated strongly-typed route helpers for
+  package:go_router
+version: 1.0.0
+repository: https://github.com/flutter/packages/tree/main/packages/go_router_builder
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+go_router_builder%22
+
+environment:
+  sdk: ">=2.14.0 <3.0.0"
+
+dependencies:
+  analyzer: '>=2.7.0 <4.0.0'
+  async: ^2.8.0
+  build: ^2.0.0
+  build_config: ^1.0.0
+  collection: ^1.14.0
+  meta: ^1.7.0
+  path: ^1.8.0
+  path_to_regexp: ^0.4.0
+  source_gen: ^1.0.0
+  source_helper: ^1.3.0
+
+dev_dependencies:
+  all_lint_rules_community: ^0.0.4
+  build_runner: ^2.0.0
+  # This is a "fake" package to enable testing
```

Comment:
what happen if it depends on the real package?

Reply:
We cannot because of how build work w/ Flutter. the build system cannot access anything that depends on Flutter in tests. This is the only way to test!